server/runtime/src/main/java/org/infinispan/server/Bootstrap.java

```diff
@@ -100,6 +100,13 @@ protected void handleArgumentCommand(String command, String parameter, Iterator<
             // Fall through
          case "--port-offset":
             properties.setProperty(Server.INFINISPAN_PORT_OFFSET, parameter);
+            int offset = Integer.parseInt(parameter);
+            if (!properties.containsKey(Server.JGROUPS_BIND_PORT)) {
+               properties.setProperty(Server.JGROUPS_BIND_PORT, Integer.toString(Server.DEFAULT_JGROUPS_BIND_PORT + offset));
```

Comment:
JGroups increments the bind_port if it isn't available. Is this a user request? what's the use case?